waku.nimble

```diff
@@ -107,4 +107,8 @@ task chat2bridge, "Build chat2bridge":
 ### Waku Tooling
 task wakucanary, "Build waku-canary tool":
   let name = "wakucanary"
-  buildBinary name, "tools/wakucanary/", "-d:chronicles_log_level=TRACE"
+  buildBinary name, "tools/wakucanary/", "-d:chronicles_log_level=TRACE -d:chronicles_runtime_filtering:on"
+
+task networkmonitor, "Build network monitor tool":
+  let name = "networkmonitor"
+  buildBinary name, "tools/networkmonitor/", "-d:chronicles_log_level=TRACE -d:chronicles_runtime_filtering:on"
```

Comment:
The `-d:chronicles_runtime_filtering:on"` should be specified in the `nim.cfg` next to the tool/app binary. Check the other binaries' `nim.cfg` for reference

Reply:
sure, added it in 725cd9f7c41b502f0c21f6b0a3f0f3c110c72f83

The thing I don't like is that you don't get in the compiling logs the flags you are using in `nim.cfg`, which imho can lead to problems. Increasing `verbosity` the max I managed to see was this. used [this](https://nim-lang.org/docs/nimc.html#compiler-usage-configuration-files) as reference.
```console
Hint: used config file '/Users/alrevuelta/Github/nwaku/vendor/nimbus-build-system/vendor/Nim/config/nim.cfg' [Conf]
Hint: used config file '/Users/alrevuelta/Github/nwaku/vendor/nimbus-build-system/vendor/Nim/config/config.nims' [Conf]
Hint: used config file '/Users/alrevuelta/Github/nwaku/config.nims' [Conf]
Hint: used config file '/Users/alrevuelta/Github/nwaku/tools/networkmonitor/nim.cfg' [Conf]
```

Unless I'm missing something `chronicles_runtime_filtering` is not present in the logs while being a flag, which imho is dangerous. Since I like being explicity, I prefered to have it directly in the makefile.

But note that I adapted the code to your suggestion following the repo "pattern".